includes/definitions/ge-digitalenergy.yaml

```diff
@@ -0,0 +1,16 @@
+os: ge-digitalenergy
```

Comment:
Is this the best name for the os? It's usually based on the firmware or actual OS the device runs like IOS, JunOS, etc. If it is then that's not an issue.